Moving-an-object/test.js

```diff
@@ -0,0 +1,36 @@
+const staticBlock = document.querySelector(".static-block");
```

Comment:
```suggestion
const staticBox = document.querySelector(".static-block");
```
Prefer `box` instead of `block` in names